sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusReceiverOptions.cs

```diff
@@ -41,6 +41,11 @@ public int PrefetchCount
         /// </summary>
         public SubQueue SubQueue { get; set; } = SubQueue.None;
 
+        /// <summary>
```

Comment:
nit:  Given that this is `public`, you probably want to consider filling this out.

Reply:
Yep, I didn't do a full doc pass as I won't be merging this until next week at the earliest.